app/reducers/gameState.js

```diff
@@ -60,11 +61,21 @@ export const gameState = (state = baseState, action) => {
         return state;
       }
     case 'END_TURN':
+      const prestigeEarned = party.reduce(
+        (acc, table) =>
+          acc +
+          table.reduce((tableTotal, { prestige }) => tableTotal + prestige, 0),
+        0,
+      );
       const leaving = party.shift();
       guestDiscard = state.guestDiscard.concat(leaving);
       party[5] = [];
-      partyPool.filter(c => !leaving.includes(c));
```

Comment:
This line seemed to be a no-op. The intent seemed to be that any guest card being discarded is also no longer buyable?

Reply:
partyPool is a "flat" representation of all members at your party.  I wanted to be able to fold over your current party attendees without going down the two-dimensional array of each table.

I use partyPool in ADD_GUEST to determine if the guest you are adding is coming from the guestDeck (being bought) or from the _existing_ party pool (i.e., being moved from one table to another).  As such, when you end your turn and people leave the party, so too must I prune this one-dimensional representation of your current attendees.  

That said, am I doing that correctly here?  Moreover, my github-fu is weak.  If we decide to keep this line, what is that workflow, does this stay in review until more changes are made?

Reply:
>That said, am I doing that correctly here?

`filter` doesn't mutate the receiver, so you'd need to assign the result to a var for it to persist (APIs that mix mutation and return values are super confusing 😒 ).

But! If you want to keep the behavior, I typically respond to a review with Request Changes (should be top right of the PR screen). Give it a shot and I'll respond with an update 😄